src/lib/datepicker/calendar.ts

```diff
@@ -29,7 +30,39 @@ import {MatDatepickerIntl} from './datepicker-intl';
 import {MatMonthView} from './month-view';
 import {MatMultiYearView, yearsPerPage} from './multi-year-view';
 import {MatYearView} from './year-view';
+import {ComponentPortal, ComponentType, Portal} from '@angular/cdk/portal';
+import {Subject} from 'rxjs/Subject';
+import {takeUntil} from 'rxjs/operators/takeUntil';
 
+/** Default header for MatCalendar */
+@Component({
+    moduleId: module.id,
+    selector: 'mat-calendar-header',
+    templateUrl: 'calendar-header.html',
+    encapsulation: ViewEncapsulation.None,
+    preserveWhitespaces: false,
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class MatCalendarHeader implements OnDestroy {
+  /** Subject that emits when the component has been destroyed. */
+  private _destroyed = new Subject<void>();
+
+  private _intlChanges: Subscription;
+
+  constructor(@Host() public calendar: MatCalendar<any>,
+              private _intl: MatDatepickerIntl,
```

Comment:
remove `private` since this is not used outside the constructor

Reply:
done in https://github.com/angular/material2/pull/9639/commits/4c6646708cd43c5acd2ed1af81ec0adf1f33300a